### PR TITLE
[PaddleAudio][Features]Add audio features.

### PR DESCRIPTION
**Pull request by KPatr1ck**
### PR types

New features

### PR changes

APIs

### Describe

- Add MFCC
- Add kaldi feature.

### CODE REVIEW

File: paddleaudio/features/spectrum.py [hunk 1]
```diff
@@ -459,3 +479,23 @@ def forward(self, x):
...
             top_db=self.top_db)
         return log_mel_feature
+
+
+class MFCC(nn.Layer):
+    def __init__(self,
+                 sr: int=22050,
+                 n_mfcc: int=40,
+                 norm: str='ortho',
+                 **kwargs):
```
Comment (zh794390558) [inline paddleaudio/features/spectrum.py:501]: api需要参数显示指出

File: paddleaudio/features/spectrum.py [hunk 2]
```diff
@@ -238,6 +239,20 @@ def power_to_db(magnitude: paddle.Tensor,
     return log_spec
 
 
+def create_dct(n_mfcc: int, n_mels: int, norm: str='ortho'):
```
Comment (zh794390558) [inline paddleaudio/features/spectrum.py]: norm  Optional[str]

**Comment by KPatr1ck** [comment]
增加Kaldi的fbank和mfcc，待测试数值对齐。

**Comment by KPatr1ck** [comment]
https://github.com/PaddlePaddle/PaddleSpeech/pull/1494

**Pull request closed without merging**